doc/usage.md

```diff
@@ -0,0 +1,114 @@
+- [Bazel configs](#bazel-configs)
```

Comment:
I imagine this will be expanded to include much more than configs right?

Reply:
Correct. This is just a section that needs to be in this guide, so I thought I would add it alongside this PR.